packages/arb-avm-cpp/data_storage/src/arbcore.cpp

```diff
@@ -999,6 +1011,17 @@ ArbCore::loadLastMatchingMachine(
     return rocksdb::Status::NotFound();
 }
 
+std::unique_ptr<MachineThread> ArbCore::getMachineThreadFromSimpleCheck(
```

Comment:
I think the code would actually be more readable with this inlined since it looks like it's only used in one place

Reply:
:+1: